src/Illuminate/Support/Collection.php

```diff
@@ -280,7 +280,7 @@ public function dd()
     /**
      * Dump the collection.
      *
-     * @return void
+     * @return $this
```

Comment:
No it shouldn't, look at `each`, `forget`, `prepend`, `push`, `put`, `tap` and `transform`.